### DISCUSSION TITLE
modbus Self-Use calculate or register?

### DISCUSSION THREAD

**Discussion by prodigy9999** [wills106/homeassistant-solax-modbus#896] [General]
In the Cloud app I can also see the value Self-Use (total per day or monthly. 
How can I get this value over modbus? register?
How ist this calculated?

**Comment by wills106** [comment]
What Inverter are you using?

If SolaX does any of the disabled by default sensors match? Such as "E Charge To..."?

**Comment by prodigy9999** [comment]
X3-Hybrid-G4
How do you calculate it in your integration? perhaps it is the same like "solax house load"?

**Comment by prodigy9999** [comment]
Perhaps you can help, I can't find it anywhere.
